compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/FunctionReferenceLowering.kt

```diff
@@ -398,6 +398,7 @@ internal class FunctionReferenceLowering(private val context: JvmBackendContext)
 
             body = context.createJvmIrBuilder(symbol).run {
                 var unboundIndex = 0
+                at(this@createFunctionReferenceInvokeMethod.startOffset, this@createFunctionReferenceInvokeMethod.startOffset)
```

Comment:
Maybe pass them to the `createJvmIrBuilder` call two lines above?

Also, why `startOffset, startOffset` instead of `startOffset, endOffset`?

Reply:
Ah, didn't realize you could pass it to `createJvmIrBuilder`, thank you! Fixed to used `startOffset, endOffset`.

Reply:
Updated #3725 as well, there we really do want to use the start offset for both start and end (as this is the first line of the class that we want to use for the entire bridge body). I think that is why I ended up using `startOffset,startOffset` in this change as well. Cleaned up now! Thanks!